nf_core/module-template/modules/functions.nf

```diff
@@ -9,6 +9,13 @@ def getSoftwareName(task_process) {
     return task_process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()
```

Comment:
is `getSoftwareName` still needed for anything?

Reply:
I think you're right. It's redundant now. Removing it.

Reply:
Whoops. It's sometimes used as a publish_dir parameter.
Using the process name should be fine though right?